build/icons-page.js

```diff
@@ -4,7 +4,7 @@ const path = require('path')
 const iconsPath = path.join(__dirname, '..', 'icons')
 const iconsList = fs.readdirSync(iconsPath)
 const iconsData = iconsList.map(filename => {
-  const url = `icons/${filename}`
+  const url = `../icons/${filename}`
```

Comment:
This change is breaking the build: if you run `npm run -s deploy:docs` and then open the resulting CID, you'll see the icons can't be visualized. It works when opening ìndex.html` manually because the icons are, in fact, at that relative location, but it doesn't work on the final version.

Reply:
Thanks for this -- original code was referencing an `icons` directory inside the `docs` directory but it wasn't being auto-generated on `npm run build`. Thought I could shortcut it by just referring to the master icons directory, but nope; in absence of that added a command to `run build` that will copy icons into `docs/icons` with the latest from the master icons directory. See https://github.com/ipfs-shipyard/ipfs-css/pull/40/commits/041042d5b0062e343335bf38c43b705b94417810